DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/DelegateProgress.cshtml

```diff
@@ -0,0 +1,64 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.CourseDelegates
+@using Microsoft.Extensions.Configuration
+@model DelegateProgressViewModel
+
+@{
+  ViewData["Title"] = "Delegate progress";
+  ViewData["Application"] = ApplicationType.TrackingSystem.Name;
+  ViewData["HeaderPath"] = ApplicationType.TrackingSystem.HeaderPath;
+  ViewData["HeaderPathName"] = ApplicationType.TrackingSystem.HeaderPathName;
+  var routeParamsForBackLink = new Dictionary<string, string?> {
+    {
+      "customisationId", Model.CustomisationId.ToString()
+    }
+  };
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="_CourseDelegatesBreadcrumbs" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-heading-xl">@ViewData["Title"]</h1>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-one-quarter nhsuk-body-l">
+        Delegate
+      </div>
+      <div class="nhsuk-grid-column-three-quarters nhsuk-body-l nhsuk-u-font-weight-bold word-break">
+        @Model.DelegateNameAndEmail
+      </div>
+    </div>
+
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-one-quarter nhsuk-body-l">
+        Course
+      </div>
+      <div class="nhsuk-grid-column-three-quarters nhsuk-body-l nhsuk-u-font-weight-bold">
+        @Model.CourseName
+      </div>
+    </div>
+
+    @if (!Model.IsCourseActive) {
+      <partial name="_InactiveCourseInset" />
+    }
+
+    <partial name="_DelegateProgressSummary" model="Model" />
+
+    <a class="nhsuk-button left-button-mobile-margin-bottom nhsuk-u-margin-right-2" role="button">
+      View detailed progress
+    </a>
+    <a class="nhsuk-button nhsuk-button--secondary" role="button">
+      View learning log
+    </a>
+
+    <vc:back-link asp-controller="CourseDelegates" asp-action="Index" link-text="Go back" asp-all-route-data="@routeParamsForBackLink" />

```

Comment:
Is this pointing back to a paginated list? If so, we should send the user back to the page they came from. Speak to David about that - he's making some changes to the back link to allow that in his current ticket.

Reply:
It is indeed pointing back to a paginated list. At least it should be, but pagination is not currently set up on that page yet, we are just grabbing a list of the first 10 delegates for the courses. As implementing the pagination is out of scope on the 621 ticket, I've added a section to the description of the ticket where it is in scope to also change the back link to work correctly: https://softwiretech.atlassian.net/browse/HEEDLS-564